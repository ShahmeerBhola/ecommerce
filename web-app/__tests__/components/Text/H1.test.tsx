import React from 'react';
import renderer from 'react-test-renderer';
import H1 from '../../../components/Text/H1';

describe('H1 component', () => {
  test('it renders properly with default color', () => {
    const tree = renderer.create(<H1>Header</H1>).toJSON();
    expect(tree).toMatchSnapshot();
  });

  test('it renders properly with non-default color', () => {
    const tree = renderer.create(<H1 color="red-200">Header</H1>).toJSON();
    expect(tree).toMatchSnapshot();
  });
});
