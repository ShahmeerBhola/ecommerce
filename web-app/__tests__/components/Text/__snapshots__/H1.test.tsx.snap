// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`H1 component it renders properly with default color 1`] = `
<h1
  className="font-primary font-bold text-black text-3xl uppercase"
>
  Header
</h1>
`;

exports[`H1 component it renders properly with non-default color 1`] = `
<h1
  className="font-primary font-bold text-red-200 text-3xl uppercase"
>
  Header
</h1>
`;
